cmd/workspace.go

```diff
@@ -198,6 +203,85 @@ func newWorkspaceUserListCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+/*
+	Workspace teams commands
+*/
+
+func newWorkspaceTeamRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "team",
+		Short: "Manage Workspace Team resources",
+		Long:  "Teams can be added or removed from Workspaces",
+	}
+	cmd.AddCommand(
+		newWorkspaceTeamAddCmd(out),
+		newWorkspaceTeamUpdateCmd(out),
+		newWorkspaceTeamRmCmd(out),
+		newWorkspaceTeamListCmd(out),
+	)
+	return cmd
+}
+
+func newWorkspaceTeamAddCmd(out io.Writer) *cobra.Command {
```

Comment:
newWorkspaceTeamCreateCmd

Reply:
If a team already exists (from IDP) but needs to be added to a workspace. Do we still need a refactor here?